Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -274,9 +274,9 @@ return
 							Type = "boolean",
 						},
 					},
-					Notes = "Returns whether the specified block type can hold snow atop.",
+					Notes = "Returns whether the specified block type lets rain pass through.",
```

Comment:
So an `IsRainBlocker` is true if it lets rain pass through, and false if it **blocks** the rain? Where's the logic in that?